packages/engine/src/interaction/components/GrabbableComponent.ts

```diff
@@ -33,7 +37,16 @@ import { Entity } from '@etherealengine/ecs/src/Entity'
 export const GrabbableComponent = defineComponent({
   name: 'GrabbableComponent',
   jsonID: 'EE_grabbable', // TODO: rename to grabbable
-  toJSON: () => true
+
+  toJSON: () => true,
+
+  reactor: function () {
+    const entity = useEntityContext()
+    useEffect(() => {
+      setComponent(entity, RigidBodyComponent, { type: BodyTypes.Kinematic })
```

Comment:
we only want to make rigidbodies kinematic if when they are grabbed, and not do it if there is not a rigidbody component

Reply:
removing this logic completely, did not realize that grabbables are meant to function even in the absence of a RigidBodyComponent